x-pack/plugins/endpoint/public/applications/endpoint/view/alerts/resolver.tsx

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import React from 'react';
+import styled from 'styled-components';
+import { Provider } from 'react-redux';
+import { CoreStart } from 'kibana/public';
+import { Resolver } from '../../../../embeddables/resolver/view';
+import { storeFactory } from '../../../../embeddables/resolver/store';
+
+export const AlertDetailResolver = styled(
+  React.memo(
+    ({
+      className,
+      selectedEvent,
+      coreStart,
+    }: {
+      className?: string;
+      selectedEvent: object;
```

Comment:
❔ 2 question: is selectedEvent being used (or will it)? can it be a ResolverEvent type?